src/rules/javascript/eval_string_arg.js

```diff
@@ -0,0 +1,16 @@
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      if (node.callee.property.type === 'Identifier' &&
+          (node.callee.property.name === 'setTimeout' ||
+           node.callee.property.name === 'setInterval')) {
+
+        if (node.arguments.length > 0 &&
+            (node.arguments[0].type !== 'FunctionExpression') &&
```

Comment:
Yeah I like this way more too; if we started checking for more than two either a helper or `indexOf` might be nicer, but this is best for now.